lab_1/main.py

```diff
@@ -4,39 +4,75 @@
 """
 
 
+def checker(con_object, type_ob):
+    if isinstance(con_object, type_ob) and type_ob is list:
+        length = len(con_object) > 0
+        bull_value = None not in con_object and bool not in con_object
+
+        if length and bull_value:
+            return True
+    elif isinstance(con_object, type_ob) and type_ob is str and len(con_object) != 0:
+        return True
+    elif isinstance(con_object, type_ob) and type_ob is int and str(con_object) != "True":
+        return True
+
+    return False
+
+
 def tokenize(text: str) -> list:
     """
-    Splits sentences into tokens, converts the tokens into lowercase, removes punctuation
-    :param text: the initial text
-    :return: a list of lowercased tokens without punctuation
-    e.g. text = 'The weather is sunny, the man is happy.'
-    --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
-    """
-    pass
+      Splits sentences into tokens, converts the tokens into lowercase, removes punctuation
+      :param text: the initial text
+      :return: a list of lowercased tokens without punctuation
+      e.g. text = 'The weather is sunny, the man is happy.'
+      --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
+      """
+    tokens = []
+    if not isinstance(text, str):
+        return []
+
+    for token in text.lower().split():
+        token = "".join([letter for letter in token if letter.isalpha()])
+        if token != "":
+            tokens.append(token)
+    return tokens
+
+
+def get_stop_words() -> list:
+    with open("stop_words.txt") as file:
```

Comment:
Define a var `path_to_stop_words` like:
`current_dir = os.path.dirname(os.path.abspath(__file__))`
`path_to_stop_words = os.path.join(current_dir, 'stop_words.txt')`

And open it.